openedx/features/caliper_tracking/tests/expected/edx.team.changed.json

```diff
@@ -0,0 +1,43 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "Modified",
+  "actor": {
+    "id": "http://localhost:18000/u/honor",
+    "name": "honor",
+    "type": "Person"
+  },
+  "eventTime": "2018-12-27T09:41:40.334Z",
+  "extensions": {
+    "extra_fields": {
+      "accept_language": "en-US,en;q=0.9",
+      "agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.80 Safari/537.36",
+      "event_source": "server",
+      "event_type": "edx.team.changed",
+      "host": "8c3b43aa7b1b",
+      "ip": "172.18.0.1",
+      "page": null,
+      "path": "/api/team/v0/teams/team1-00f9ce77ea4a4f71bcf775857df8e34e",
+      "session": "8b5aaa43369a0d612918005f1a65f27f",
+      "user_id": 2
+    }
+  },
+  "id": "urn:uuid:d4618c23-d612-4709-8d9a-478d87808067",
+  "object": {
+    "extensions": {
+      "course_id": "course-v1:edx+CS0110+2014_T",
```

Comment:
Don't you think we must stay consistent with the rest of team ? Move this 'course_id' and 'org_id' to main extensions in all of your events.

Reply:
I believe it doesn't make that much of a difference. Also, if this information is relevant to the object then it should be moved.

Reply:
It doesn't make any difference but i think consistency matter. What if we have to write anything in future based on our this code and we need 'course_id' we can extract it easily by going to ['extensions']['extra_fields'] but on this event's it will fail.

Reply:
When that time comes we will worry about it then. Right now it doesn't matter.